paper/src/main/java/vg/civcraft/mc/civmodcore/world/locations/chunkmeta/ChunkCoord.java

```diff
@@ -72,19 +73,19 @@ void persistPlugin(short id) {
 			persistChunkMeta(chunkMeta);
 		}
 	}
-	
+
 	private static void persistChunkMeta(ChunkMeta<?> chunkMeta) {
 		switch (chunkMeta.getCacheState()) {
-		case NORMAL:
-			break;
-		case MODIFIED:
-			chunkMeta.update();
-			break;
-		case NEW:
-			chunkMeta.insert();
-			break;
-		case DELETED:
-			chunkMeta.delete();
+			case NORMAL:
+				break;
+			case MODIFIED:
+				chunkMeta.update();
+				break;
+			case NEW:
+				chunkMeta.insert();
+				break;
+			case DELETED:
+				chunkMeta.delete();
```

Comment:
Default case for unintended values is missing, this might lead to strange issues.

Reply:
Agree, fixed.
```
			case DELETED:
				chunkMeta.delete();
			default:
				throw new IllegalArgumentException("Unsupported cache state '" + chunkMeta.getCacheState() + "'");
```

Thanks for the review!